ansible/roles/base/templates/etc/ssh/sshd_config.j2

```diff
@@ -60,7 +60,7 @@ StrictModes yes
 Subsystem sftp /usr/lib/openssh/sftp-server
 SyslogFacility AUTH
 TCPKeepAlive no
-UseDNS yes
+UseDNS no
```

Comment:
QUESTION: What does this mean?

Reply:
This does controls whether sshd does a reverse DNS lookup when you log in. It slows down login and is only really needed if you control access using hostname. See https://unix.stackexchange.com/a/56947 and the answer after that one.